Sources/Basics/FileSystem/FileSystem+Extensions.swift

```diff
@@ -466,6 +467,14 @@ extension FileSystem {
         }
     }
 
+    public func getOrCreateDotSwiftPMDirectory() throws -> AbsolutePath {
```

Comment:
You won't need this separate function if you follow the pattern for the rest of the directories in `~/.swiftpm` defined here, each of them already calls `createDirectory(dotSwiftPM, recursive: true)`.